resources/views/layouts/tables.blade.php

```diff
@@ -0,0 +1,68 @@
+@extends ('layouts.app')
+
+@yield ('variables')
+
+@section ('script')
+
+<script type='text/javascript' src='/DataTables/media/js/jquery.dataTables.min.js'></script>
+<script type='text/javascript' src='/DataTables/extensions/Buttons/js/dataTables.buttons.min.js'></script>
+<script type='text/javascript' src='/DataTables/extensions/Buttons/js/buttons.colVis.js'></script>
+
+<script type='text/javascript'>
+	var table;
+
+	$(document).ready(function(){
+	    table = $('#table').DataTable( {
```

Comment:
The space before this curly brace is odd (as well as the space after the ending curly brace).
Is it something that you think we should do?